docs/docs/examples/guides/object-detection-example.md

```diff
@@ -1,91 +1,199 @@
+
+```shell
+git clone https://github.com/aica-technology/docker-images.git && cd docker-images
+```
+
+then:
+
+```shell
+cd yolo_model_converter
+./build-run.sh
+```
+
+By default, this will download and convert `yolo12n` for you. If you wish to specify one of the other models that
+Ultralytics is offering, simply specify it as follows:
+
+```shell
+./build-run.sh --model yoloZZZZ
+```
+
+:::note
+
+If you have `ultralytics` installed in your Python environment or you are willing to install it, then download any of
+the models available [here](https://github.com/sunsmarterjie/yolov12). For the purposes of this example, you can opt
+for a smaller model such as the `yolo12n`.
 
-To convert a PT file to ONNX, run the following Python code with ultralytics installed:
+To convert a `.pt` file to `.onnx`, run the following Python code with `ultralytics` installed:
 
 ```python
 from ultralytics import YOLO
-model = YOLO("yolov12n.pt")
-model.export(format="onnx")  # creates 'yolov12n.onnx'
+model = YOLO("yolo12n.pt")
+model.export(format="onnx")  # creates 'yolo12n.onnx'
 ```
 
-For this example, also download the standard coco.yaml class
-file [here](https://github.com/ultralytics/ultralytics/blob/main/ultralytics/cfg/datasets/coco.yaml).
-Move the ONNX model file and the YAML class file into a new directory.
+This approach is also necessary if you are using custom models instead of the ones distributed by Ultralytics.
+:::
+
+### Class file
 
-In AICA Launcher, create a configuration with the following core version and packages:
+Along with the YOLO model from the previous paragraph, you will also need a **class file** (for example, `coco.yaml`)
+that maps the numeric class IDs predicted by the model to their corresponding class names. This file is specific to the
+model you are using and is typically defined during training, meaning that the mapping between class IDs and labels is
+fixed once the model is trained.
+
+Modifying the class file after training will not affect the model's behavior or predictions. It will only change the
+text labels displayed in your annotations. As a result, manual edits are generally safe but may lead to semantic
+inconsistencies if the names no longer match the model's intended classes.
+
+For the purposes of this example, download the standard `coco.yaml` class
+file [here](https://github.com/ultralytics/ultralytics/blob/main/ultralytics/cfg/datasets/coco.yaml) and move it to your
+`data` folder, where you also stored your YOLO model.
 
-- AICA Core v4.4.2
-- `collections/object-detection` for the YOLO Executor component (TODO: check package name post-release and specify
-  version)
-- `components/core-vision v1.0.0` for the Camera Streamer component
+## AICA Launcher configuration
 
-Under Advanced Settings, add a volume containing the folder where you stored your YOLO model files and link it to
-`/files`.
+In AICA Launcher, create a configuration with the following core version and packages:
```

Comment:
I feel that here we need to make the link to the data folder more explicit, such that users don't forget to choose that data folder path or add the volume